components/contact-list/src/ContactList.svelte

```diff
@@ -172,15 +173,17 @@
   */
   function setContacts() {
     status = "loading";
-    ContactStore.addContacts(query, offset, _this.contacts_to_load).then(
-      (results?: Contact[]) => {
-        if (results && results.length > 0) {
-          hydratedContacts = $ContactStore[queryKey] ?? results;
-          offset += _this.contacts_to_load;
-        }
-        status = "loaded";
-      },
-    );
+    ContactStore.addContacts(query, {
+      offset,
+      limit: _this.contacts_to_load,
+      ...(_this.group ? { group: _this.group } : {}),
+    }).then((results?: Contact[]) => {
```

Comment:
```suggestion
    ContactStore.addContacts(query, `limit=${ _this.contacts_to_load}&offset=${offset}&group=${group}`
    ).then((results?: Contact[]) => {
```

Reply:
For this to work you'd have to add a variable like what you have here `$: queryString = { offset, limit: _this.contacts_to_load, ...(_this.group ? { group: _this.group } : {})};`